app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    todo, file_from, file_to, *_ = command.split() + [None] * 2
```

Comment:
For what you need `[None] * 2` ?

Reply:
Oh, it should be [None] * 3
If split will return insufficient len list, `file_from`, `file_to` will be None
Thus user can pass an empty string or just mv in the function